frontend/src/components/Input.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ChangeEvent } from "react";
+import { useState } from "react";
+interface InputProps {
```

Comment:
Add an empty line between the imports and the `InputProps` interface. Also, replace both the imports with this line: 

`import { ChangeEvent, useState } from "react";`